### PR TITLE
Fix/787

### LINKED ISSUE #787: Fix all sandbox/ files to use 'quality' instead of 'accuracy' for FASTQ.
This needs to be done before next release, for sure.

### PR DESCRIPTION
**Pull request by aditi9783**
Replaced 'accuracy' with 'quality' in sandbox/ files. Fixes #787.

### CODE REVIEW

**Comment by aditi9783** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,7 @@
+2015-02-23	Aditi Gupta	<agupta@msu.edu>
+
+   * sandbox/collect-reads.py, correct-errors.py, normalize-by-median-pct.py, slice-reads-by-coverage.py, sweep-files.py, sweep-reads3.py, to-casava-1.8-fastq.py: Replaced 'accuracy' with 'quality'. Fixes #787.
```
Comment (ctb) [inline ChangeLog]: Please shorten line to < 80 char

Comment (ctb) [inline ChangeLog]: also use notation: sandbox/{collect-reads.py,correct-errors.py,...}

**Comment by ctb** [comment]
add to test list

**Comment by ctb** [comment]
add to testlist

**Comment by aditi9783** [comment]
Ready to review.

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,10 @@
+2015-02-23	Aditi Gupta	<agupta@msu.edu>
```
Comment (mr-c) [inline ChangeLog]: Just two spaces between the date and your name and just two spaces between your name and your email address.

File: ChangeLog [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+
+   * sandbox/{collect-reads.py, correct-errors.py,
```
Comment (mr-c) [inline ChangeLog]: This paragraph should be indented by one more column to match the formatting of the rest of the ChangeLog entries.

**Comment by mr-c** [comment]
Otherwise it looks good!

**Comment by aditi9783** [comment]
Ready to review.

**Comment by mr-c** [comment]
LGTM. As soon as this is mergeable I shall do so.

**Comment by mr-c** [comment]
Thanks!

**Comment by ctb** [comment]
Just in case you're wondering, khmer-recipes works with this.

**Pull request merged**